var/spack/repos/builtin/packages/emboss/package.py

```diff
@@ -38,3 +38,9 @@ class Emboss(AutotoolsPackage):
     depends_on('libxpm')
     depends_on('libgd')
     depends_on('postgresql')
+
+    def install(self, spec, prefix):
+        # Delete $(bindir)/embossupdate to skip update checks
+        makefile = FileFilter('Makefile')
+        makefile.filter('$(bindir)/embossupdate', '', string=True)
+        make('install')
```

Comment:
I would do:
```python
@run_after('configure')
def skip_update_checks(self):
    # Delete $(bindir)/embossupdate to skip update checks
    filter_file('$(bindir)/embossupdate', '', 'Makefile', string=True)
```
That way you don't need to override any phases, you patch between phases.

Reply:
fixed.